src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/GetSasToken.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator
+
+import org.slf4j.LoggerFactory
+import java.net.URLEncoder
+import java.util.Base64
+import javax.crypto.Mac
+import javax.crypto.spec.SecretKeySpec
+
+private val LOG = LoggerFactory.getLogger(GetSasToken::class.java)
+
+class GetSasToken {
```

Comment:
same here any reason we are adding more Kotlin files ?

Reply:
Its a simple createSasToken cli utility ... could have written it in python too.
I think its better to have it than not  :D.
Does it matter which language our utility apps are written in ... i.e. bash, python, kotlin, groovy ?

Reply:
why not java ? point is we are mixing two different languages.

Reply:
sure deleted